fmt/src/git.rs

```diff
@@ -141,7 +141,7 @@ pub fn resolve_file_attrs(repo: &Repository) -> anyhow::Result<HashMap<String, G
                 Ok::<_, Infallible>(Default::default())
             },
         )?;
-        prev_commit = this_commit;
+        prev_tree = tree;
         cache.clear_resource_cache();
     }
 
```

Comment:
These changes seem no help. Because we still find object for tree? And iter over all the changes on every location.

Reply:
For https://github.com/korandoru/hawkeye/pull/150/commits/e576dad548bd1ff6e3163da3b2ab662c7b76ca53